### PR TITLE
Examples

### PR DESCRIPTION
**Pull request by mudler**
See https://github.com/rancher/elemental-toolkit/issues/1402

### CODE REVIEW

File: docs/content/en/docs/Examples/cloud_config.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            BOOTPROTO='static'
+            IPADDR='192.168.1.2/24'
+         permissions: 0600
+         owner: 0
+         group: 0
+```
+
+### Static IP
+
+Set static IP with wicked to an additional NIC:
```
Comment (fgiudici) [inline docs/content/en/docs/Examples/cloud_config.md:40]: Not sure if we want to use eth0 here to differentiate from the "Additional nick" section above... 

```suggestion
Set static IP with wicked:
```

File: docs/content/en/docs/Examples/cloud_config.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+```yaml
+name: "Default network configuration"
+stages:
+   boot:
+     - commands:
+       - wicked ifup eth1
```
Comment (fgiudici) [inline docs/content/en/docs/Examples/cloud_config.md:47]: Matching line 51
```suggestion
       - wicked ifup eth0
```

**Review by fgiudici — APPROVED** [review]
LGTM!

**Comment by mudler** [comment]
reworked it slightly, also tested locally :+1:

**Review by Itxaka — APPROVED** [review]

File: docs/content/en/docs/Examples/cloud_config.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+## Networking with wicked
+
+By default all interfaces will get automatically an IP address, however, there are situations where a static IP is desired, or custom configuration to be specified, here you can find some network settings with wicked.
+
+### Additional NIC
+
+Set static IP to an additional NIC:
+
+```yaml
+name: "Default network configuration"
```
[the remark this answers is not in the document]
Comment (mudler) [reply docs/content/en/docs/Examples/cloud_config.md:23]: right.. we could do syntax checking that's sure. We could even have a test suite during integration just for example files.. although it's tricky, as in this case would require specific network setup

[the remark this answers is not in the document]
Comment (mudler) [reply docs/content/en/docs/Examples/cloud_config.md:23]: maybe a simple job calling elemental cloud-init ... to see if those breaks? that's doable

[the remark this answers is not in the document]
Comment (Itxaka) [reply docs/content/en/docs/Examples/cloud_config.md:23]: in that case, the example should move to elemental-cli maybe? Maybe its the proper place as the implementation is "done there" so the examples should be in there...

**Pull request merged**